zerver/openapi/zulip.yaml

```diff
@@ -12106,27 +12106,17 @@ paths:
                             - $ref: "#/components/schemas/UserBase"
                             - additionalProperties: false
                               properties:
+                                user_id: {}
+                                delivery_email: {}
                                 email: {}
-                                is_bot: {}
-                                avatar_url:
-                                  nullable: true
-                                avatar_version: {}
                                 full_name: {}
-                                is_admin: {}
-                                is_owner: {}
-                                is_billing_admin: {}
-                                role: {}
-                                bot_type:
-                                  nullable: true
-                                user_id: {}
-                                bot_owner_id:
-                                  nullable: true
+                                date_joined: {}
                                 is_active: {}
+                                is_owner: {}
+                                is_admin: {}
                                 is_guest: {}
-                                timezone: {}
-                                date_joined: {}
-                                delivery_email: {}
-                                profile_data: {}
+                                is_billing_admin: {}
+                                is_bot: {}
                                 is_system_bot:
```

Comment:
So, even though `is_system_bot` is inserted in the order we would want it listed in the documentation, it's rendered after all the `UserBase` properties.

I've noticed this in other parts of the documentation (events that use the `EmojiReactionBase` are one that comes to mind), but haven't looked into where the issue is in our Markdown rendering process (it's in my mental notes to look into, but hasn't been a priority).

But basically, if a schema reference is being used, any additional properties will be rendered last, so I've been defaulting to adding those to the end of the listed properties in the OpenAPI documentation for consistency with how the documentation appears when rendered.